ngx-fudis/projects/ngx-fudis/src/lib/components/form/select/common/autocomplete/autocomplete.component.ts

```diff
@@ -91,47 +91,47 @@ export class SelectAutocompleteComponent implements OnChanges, OnInit {
   /**
    * Output event for input field blur
    */
-  @Output() triggerBlur = new EventEmitter<FocusEvent>();
+  @Output() handleBlur = new EventEmitter<FocusEvent>();
 
```

Comment:
We should discuss with the team, if this name change is wanted. Autocomplete component is not a public component, so it does not have to have same API naming conventions as public components. Actually prefix `trigger` was intentionally used here to differentiate it from public `handle` outputs.